feat: adding unit tests for ipsec

mohamedasifs123 opened the pull request.

sandersms reviewed (changes requested):
Please address the Go build errors

mohamedasifs123 committed:
feat: adding unit tests for ipsec

This reverts commit 93e34b60acdd2be19bbc6bc54af7f27f50202a74.

Update go.yml

Signed-off-by: Mohamed Asif <142201466+mohamedasifs123@users.noreply.github.com>

feat: adding unit tests for ipsec

This reverts commit d7ff204dd85d0f434bd2e8e8886f4321efbee4e9.

Revert "Update go.yml"

This reverts commit cf792d3b0b69f35c30d51f17aeed9ae1432a5a9a.

Update go.yml

Signed-off-by: Mohamed Asif <142201466+mohamedasifs123@users.noreply.github.com>

Update go.yml

Signed-off-by: Mohamed Asif <142201466+mohamedasifs123@users.noreply.github.com>

Update go.yml

Signed-off-by: Mohamed Asif <142201466+mohamedasifs123@users.noreply.github.com>

feat: adding unit tests for ipsec

feat: adding unit tests for ipsec

Signed-off-by: mohamedasifs123 <asifabu272@gmail.com>

feat: adding unit tests for ipsec

Signed-off-by: mohamedasifs123 <asifabu272@gmail.com>

Update go.yml

feat: adding unit tests for ipsec

Signed-off-by: mohamedasifs123 <asifabu272@gmail.com>

mohamedasifs123 commented:
Changed the test workflow to support the strongswan integration for vici to run the tests and upload the tests to codecov.

mohamedasifs123 committed:
Update go.yml

Signed-off-by: Mohamed Asif <142201466+mohamedasifs123@users.noreply.github.com>

mohamedasifs123 committed:
Update go.yml

Signed-off-by: Mohamed Asif <142201466+mohamedasifs123@users.noreply.github.com>
